Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.cs

```diff
@@ -139,6 +139,20 @@ public void Should_succeed_when_asserting_datetime_value_is_equal_to_the_same_va
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void Should_succeed_when_asserting_datetime_value_is_equal_to_the_same_nullable_value()
```

Comment:
🙃 Although the code base is inconsistent, we prefer to use the `When_scenario_it_should_something`.

Reply:
Done! I think there should be a reference on the tests naming convention. Maybe in the Contributing guide.